editor/editor_plugin.cpp

```diff
@@ -450,6 +462,12 @@ void EditorPlugin::remove_control_from_container(CustomControlContainer p_locati
 			EditorNode::get_singleton()->get_inspector_dock_addon_area()->remove_child(p_control);
 
 		} break;
+		case CONTAINER_PROJECT_SETTING_TAB_RIGHT:
```

Comment:
Nitpick, but having "LEFT" before "RIGHT" would be better to match the order used elsewhere.